Backend/src/controllers/user.controller.ts

```diff
@@ -44,13 +44,32 @@ export default class UserController {
     }
 
     static async getProfile(req: any, res: any): Promise<void> {
-        res.json({
-            createTime: req.user.createTime,
-            email: req.user.email,
-            firstname: req.user.firstname,
-            lastname: req.user.lastname,
-            role: req.user.role,
-        });
+        if(typeof(req.params.userId) === 'undefined'){
```

Comment:
ใช้ `!(req.params.userId)` น่าจะ save กว่า

https://repl.it/@nisaruj/ShockedTrimTrapezoids